application-default.properties

```diff
@@ -372,7 +372,7 @@ object.store.swift.username=test
 object.store.swift.password=test
 object.store.swift.url=http://localhost:8080
 
-packet.manager.account.name=packet-manager
+packet.manager.account.name=${s3.pretext.value:}packet-manager
```

Comment:
why do we need this, when object.store.s3.bucket-name-prefix is already configured ?

Reply:
@sowmya695 please say why we are changing the code in this PR.